tslib/src/functions/pupil-prefs/pupil-prefs.service.ts

```diff
@@ -84,7 +84,7 @@ export class PupilPrefsDataService implements IPupilPrefsDataService {
     return this.sqlService.modifyWithTransaction(requests)
   }
 
-  // TODO why not add a lookup to redis on check creation to avoid sql hits?
+  // TODO : PERF why not add a lookup to redis on check creation to avoid sql hits?
```

Comment:
👍

Reply:
i had this as one item in a PBI, but ive promoted it to its own card now